package.json

```diff
@@ -157,7 +162,7 @@
     "rollup-plugin-angular-aot": "^0.0.4",
     "run-sequence": "^1.2.2",
     "rxjs": "5.0.0-beta.12",
-    "sass-loader": "4.0.2",
+    "sass-loader": "6.0.2",
```

Comment:
I was hoping to get rid of the "stack level too deep" issue, but we can go back to 4.1.1.